src/extra.h

```diff
@@ -18,7 +18,7 @@ static Player_getArmor_t Player_getArmor = (Player_getArmor_t) 0x8fda4;
 typedef void (*offsetCords_t)(uchar *offsetData, int *x, int *y, int *z);
 static offsetCords_t offsetCords = (offsetCords_t) 0x27c98;
 
-static uchar **Item_items = (uchar **) 0x17b250;
+static uchar **Item_items = (uchar **) 0x17b250; // Comment this out if you're using MCPI-Reborn Extended!
```

Comment:
Could you move it to not be an inline comment?

Reply:
I put it after the line so if it errors it will show the comment too.